src/compiler/crystal/tools/playground/views/layout.html.ecr

```diff
@@ -14,7 +14,7 @@
   </head>
   <body class="grey lighten-3 grey-text text-darken-2">
     <header>
-      <div class="navbar-fixed">
+      <div class="navbar">
```

Comment:
Mostly because my MacBook 11" has a small screen and the banner takes up a
good % of it.

On Wednesday, March 23, 2016, Brian J. Cardiff <notifications@github.com>
wrote:

> In src/compiler/crystal/tools/playground/views/layout.html.ecr
> <https://github.com/crystal-lang/crystal/pull/2356#discussion_r57212257>:
>
> > @@ -14,7 +14,7 @@
> >    </head>
> >    <body class="grey lighten-3 grey-text text-darken-2">
> >      <header>
> > -      <div class="navbar-fixed">
> > +      <div class="navbar">
>
> I guess you prefer non fixed navbar. But other than that, any other reason
> to change this?
>
> Probably becuase when zooming the navbar is also zoomed.
> The css is pretty straight forward and changing the font-size should
> affect mostly everything in the main area.
> Maybe there should be a presenter mode with even bigger font size...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/crystal-lang/crystal/pull/2356/files/c0de781c031c2c480b345d6f681d76469dcd9ba4..c0decb35640084111262cf997dea93deab85fbd0#r57212257>
>